Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM node:5.11-wheezy
+
+ARG admin_user
+
+WORKDIR /opt/ldnode/certs
+RUN openssl genrsa 2048 > ssl-key.pem
+RUN openssl req -new -x509 -nodes -sha256 -days 3650 -key ssl-key.pem -subj '/CN=*.localhost' > ssl-cert.pem
+
+COPY . /src
+RUN cd /src; npm install
+
+WORKDIR /src/data
+RUN echo $'@prefix n0: <http://www.w3.org/ns/auth/acl#>. \n\
```

Comment:
Yeah I was going to use `COPY` but for the variable. Would be ideal if there's a workaround for that, though when #326 is fixed we won't need it anyway.